app.go

```diff
@@ -349,6 +352,11 @@ func (a *servemodule) Configure(injector *dingo.Injector) {
 	})
 }
 
+// CueConfig for the module
+func (a *servemodule) CueConfig() string {
+	return `core: serve: port: number | *3322`
```

Comment:
```suggestion
	return `core: serve: port: int | *3322`
```

Reply:
also set a constraint for the port range

Reply:
I added the constraint for valid port range, but it's not possible to declare it as `int`:
```
cue: marshal error at path core.serve.port: conflicting values ((int & >=0) & <=65535) and 1234 (mismatched types int and float)
```

Reply:
I'll look into it